src/domain.py

```diff
@@ -317,6 +317,11 @@ def domain_add(
             pass
         raise e
 
+    if dyndns and dyndns_recovery_password:
+        domain_config_set(
+            domain, "dns.registrar.recovery_password", dyndns_recovery_password
+        )
```

Comment:
I'm just starting my review but i'm not sure I get the intent here, the dyndns_recovery_password is already defined during the previous call to `domain_dyndns_subscribe` ?